test/e2e/framework/internal/unittests/cleanup/cleanup_test.go

```diff
@@ -122,7 +122,17 @@ func TestCleanup(t *testing.T) {
 	logger, _ := ktesting.NewTestContext(t)
 	klog.SetLogger(logger)
 
-	apiServer := testapiserver.StartAPITestServer(t)
+	// disable some options to save some during bootstrap
```

Comment:
Nit:

```suggestion
	// Disable some options to save some time during bootstrap.
```

"time" was missing. I also prefer full sentences with a final stop, because then it is possible to extend comments later on without having to rewrite the existing sentence.